source/pkcore/abilitypages/chi/paralyze.rst

```diff
@@ -0,0 +1,59 @@
+.. paralyze:
+############
+Paralyze
+############
+
+Description
+###########
+
+Paralyze is a Chi Ability.
+Paralyze allows the Chiblocker to paralyze entities for a short time period. GIF 
+
+
+Usage
+#####
+
+GIF TO BE ADDED
+
+Function 1: [Left Click] the target entity 
+
+
+.. tip:: After using Paralyze, be sure to follow up with your other abilities!
```

Comment:
Change to `Paralyze can easily turn the tide of a fight, be sure to follow it up with your other abilities!`